lib/icon_data.dart

```diff
@@ -28,3 +28,13 @@ class IconDataRegular extends IconData {
           fontPackage: 'font_awesome_flutter',
         );
 }
+
+// Pro only
+class IconDataLight extends IconData {
```

Comment:
Do you think we could comment these out by default as well? This won't work for non-pro users and I would prefer to keep the API as simple as possible for the majority of users.

Reply:
Sure, that's propably a good idea since most people won't need it. Let me update this and the readme real quick